### PR TITLE
zendesk doc, images, and navigation

### PR DESCRIPTION
**Pull request by Adorism**
## Change description

This is the markdown with associated images and navigation updates for a new Zendesk Integration doc. 

### Development

- [X] Application changes have been tested appropriately

### Impact

- [ ] Code follows company security practices and guidelines
- [ ] Security impact of change has been considered
- [ ] Performance impact of change has been considered
- [ ] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [ ] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [ ] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

File: pages/docs/integrations/grouparoo-zendesk.mdx [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+  </tablebody>
+</table>
+
+## Setup
+
+You will need a Zendesk account already created in order for Grouparoo start syncing data to Zendesk. You will also need an API token. When you create a new token for Zendesk you will be prompted with the option to give it a descriptive name. The token generated will not be accessible again, once you have clicked on save, so be sure to copy the token that is given to you and store it someplace secure.
+
+In Zendesk, navigate to Admin Settings (this is the Gear Icon on the left-side navigation) > Channels > API. You can create an API key and enable API access here.
+
+SendGrid API Key management can be found [here](https://app.sendgrid.com/settings/api_keys).
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-zendesk.mdx]: Oops, this is for SendGrid.

File: pages/docs/integrations/grouparoo-zendesk.mdx [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+      <td>
+        Zendesk username. This is often the email address of a user with
+        admin-level access.
+      </td>
+    </tr>
+    <tr>
+      <td>
+        <strong>API Token </strong>(required)
+      </td>
+      <td>Zendesk api token for the admin user.</td>
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-zendesk.mdx]: Nitpick:
```suggestion
      <td>Zendesk API token for the admin user.</td>
```

File: pages/docs/integrations/grouparoo-zendesk.mdx [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+
+This Destination allows you to sync your Grouparoo Records to Records in Zendesk.
+
+Companies use Zendesk to organize their information about customers in order to facilitate communication that is seamless in trajectory from sales to support and continued communication.
+
+To [create a Destination](https://www.grouparoo.com/docs/config/destinations), you make choices about what data to sync.
+
+### Sync Modes
+
+- **Sync**: Add, update, and remove Zendesk Users as needed.This option looks for new records, changes to existing records, and deletions.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-zendesk.mdx]: Nitpick: 
```suggestion
- **Sync**: Add, update, and remove Zendesk Users as needed. This option looks for new records, changes to existing records, and deletions.
```

File: pages/docs/integrations/grouparoo-zendesk.mdx [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+- **external_id:** A User’s unique identifier that relates the Source Record and the Grouparoo Property to the User Record in Zendesk.
+
+#### Optional Properties
+
+There is a list of standard User Properties but accounts with admin access in Zendesk can also add custom user fields. Once those are added, they will be available as Properties to map to using the Grouparoo Properties in the dropdown menu which represent the information you have built into your model.
+
+### Group Data
+
+In addition to mapping Grouparoo Properties to Zendesk User Properties, you can use Grouparoo Groups to apply tags to Mailchimp Users.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-zendesk.mdx]: Oops, Mailchimp

File: pages/docs/integrations/grouparoo-zendesk.mdx [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+This tagging process starts by selecting a group from the dropdown menu in the Send Group box. The Zendesk Tag name will default to the name of the Grouparoo Group you have selected, but you can edit the Mailchimp Tag name as you want it to appear. Grouparoo will not affect tags with different names that were created and applied directly through Zendesk.
```
Comment (teallarson) [inline pages/docs/integrations/grouparoo-zendesk.mdx]: Oops, Mailchimp

**Review by teallarson — COMMENTED** [review]
Couple of accidental references to other plugins and a few super minor nitpicks, but looks good otherwise!

**Review by evantahler — APPROVED** [review]

**Pull request merged**